src/assets/styles/_containers.scss

```diff
@@ -0,0 +1,33 @@
+$container-widths: (
+  // 'xs': var(--container-width-xs),
+    'xs': 19rem,
+  // 's': var(--container-width-s),
+    's': 35rem,
+  // 'm': var(--container-width-m),
+    'm': 47rem,
+  // 'l': var(--container-width-l),
+    'l': 60.5rem,
+  // 'xl': var(--container-width-xl),
+    'xl': 75rem
```

Comment:
Unlike `_spacings` values, why did you choose to not rely on HDS variables here?

Reply:
These are not yet available as variables, but they're listed in the design tokens and documentation:

- https://github.com/City-of-Helsinki/helsinki-design-system/blob/master/packages/design-tokens/tokens/breakpoint/container-width.json 
- https://hds.hel.fi/guidelines/grid

Reply:
👍